pkg/sql/backfill.go

```diff
@@ -574,7 +574,12 @@ const (
 	indexBackfill
 )
 
-// returns true when the backfill for the indexes is complete.
+// backfillIndexesSpans returns the index spans that still have to be backfilled
+// for the indexes being added as part of the first mutation enqueued on the
+// table descriptor.
+//
+// Returns nil if the backfill for the indexes is complete (mutation no longer
```

Comment:
this seems like a natural consequence of the first paragraph, so i'd just as soon remove it.

In fact, it seems to me that this function's name and signature adequately describe it, so I suggest removing the comment altogether.